src/v2/Utils/getSaleOrLotTimerInfo.ts

```diff
@@ -87,10 +89,21 @@ export const getSaleOrLotTimerInfo = (
         copy = `${parsedHours}h ${parsedMinutes}m`
       }
 
+      // less than cascade interval until close
+      else if (
+        lotGridIntervalMinutes &&
+        parsedDays < 1 &&
+        parsedHours < 1 &&
+        parsedMinutes < lotGridIntervalMinutes
+      ) {
+        copy = `${parsedMinutes}m ${parsedSeconds}s`
+        color = "red100"
+      }
+
       // <60 mins until close
       else if (parsedDays < 1 && parsedHours < 1) {
         copy = `${parsedMinutes}m ${parsedSeconds}s`
-        color = "red100"
```

Comment:
Based on the above comment about moving the conditional in here, it could be:

``` ts
if (urgencyIntervalMinutes) {
  color = parsedMinutes < urgencyIntervalMinutes) ? "red100" : "black100"
}
else {
  color = "red100"
}
```